README.md

```diff
@@ -67,6 +66,16 @@ and learn more about the major components by taking a look at
  * [Components](docs/components.md) - Detailed look at all components
 
 
+# Community
+
+If oyu got enough of code and want to speak to people, then you got a couple
+of options:
+
+* Follow us on social media [Twitter](https://twitter.com/kubevirt)
+* Ask questions and provide help on [IRC](irc://irc.freenode.net/#kubevirt)
+* Find help and discuss on the [Google Group](https://groups.google.com/forum/#!forum/kubevirt-dev)
```

Comment:
Yeah ...

So - Previously I tried to give guidance where to put what stuff (ask questions, discuss, …) now it's just lisintg the channels. That#s fine for me tho - let's see how it goes.